src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+ nodes: 3
+ pool: default
+ network: default
+ domain: cephlab.com
+ prefix: ceph
+ numcpus: 1
+ memory: 2048
+ image: fedora34
```

Comment:
?
```suggestion
 image: centos8
```

Reply:
@epuertat Centos8 does not support 9p filesystem type which is required for sharing the host folder.